src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -71,17 +78,41 @@ public void init() throws ServletException {
   // TODO: Check if URL is valid.
   public void doPost(HttpServletRequest request, HttpServletResponse response)
```

Comment:
Let's have validations in one function as suggested, this servlet looks like it could definitely work well with that.

Reply:
Done!